Misc/NEWS.d/next/Library/2021-03-11-00-31-41.bpo-42967.2PeQRw.rst

```diff
@@ -0,0 +1,3 @@
+Allow :class:`bytes` ``separator`` argument in ``urllib.parse.parse_qs`` and
+``urllib.parse.parse_qsl`` when parsing :class:`str` query strings.  Previously,
+this would cause a ``TypeError``.
```

Comment:
Please ignore this, I'm making a commit to get CI checks to sync. Currently there seems to be 3 duplicate checks for this PR .

```suggestion
this would cause a :exc:`TypeError`.
```